cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -1180,7 +1185,7 @@ private KafkaRebalanceAnnotation rebalanceAnnotation(Reconciliation reconciliati
                     KafkaRebalanceAnnotation.none : KafkaRebalanceAnnotation.valueOf(rebalanceAnnotationValue);
         } catch (IllegalArgumentException e) {
             rebalanceAnnotation = KafkaRebalanceAnnotation.unknown;
-            LOGGER.warnCr(reconciliation, "Wrong annotation value {}={} on {}/{}",
+            LOGGER.warnCr(reconciliation, "Wrong annotation value {}={} on {}/{}. The rebalance resource supports the following annotations : none, approve, refresh, stop.",
```

Comment:
Same as above: What does the `none` mean? Also, I don't think you should have space before `:`.

Reply:
As mentioned offline, `none` is not a possible value, it's just an enum value which represents no value on the annotation that was needed for the state machine. It should not be listed here.